FAQ.md

```diff
@@ -0,0 +1,25 @@
+# FAQ
```

Comment:
Thanks so much for starting this!

I think we're going to want to divide our documentation up into four main use cases:

A. Deckering (wagneering)
B. Installation / server configuration
C. Mod Development
D. Core Development

For each of these, we need to decide whether the primary documentation is in the code or on wagn/decko.org (we'll be switching to decko.org very soon, I hope).

(A) Easy, these folks may never look at GitHub, so all documentation for them should be on decko.org
(B) This is the trickiest one.  We currently have a some duplication (eg the basic installation directions appear on both), but I'm think decko.org needs to be the go-to place. We have a lot more community-generated content there (eg OS-specific stuff), there will be some installations (like UBOS) where people don't interact directly with code/github, and 
(C) probably code, though perhaps they will use it most on somewhere like this: http://www.rubydoc.info/gems/card/Card. Basically, it will be the equivalent of api.rubyonrails.org.
(D) definitely code